ipaplatform/redhat/constants.py

```diff
@@ -11,9 +11,14 @@
 from __future__ import absolute_import
 
 from ipaplatform.base.constants import BaseConstantsNamespace
+from ipaplatform.osinfo import osinfo
 
+# RHEL 7 and earlier use /etc/sysconfig/nfs
+# RHEL 8 uses /etc/nfs.conf
+HAS_NFS_CONF = osinfo.id == 'rhel' and int(float(osinfo.version_id)) >= 8
```

Comment:
Please use `osinfo.id in ('rhel', 'centos')` here instead of `==`. On current CentOS 7 you'd get 'centos' as a system ID:

```
# python
Python 2.7.5 (default, Oct 31 2018, 18:48:32) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.path.append('.')
>>> import osinfo
>>> print(osinfo.osinfo.id in ('centos', 'rhel'))
True
>>> print(osinfo.osinfo.id)
centos
```

I assume the same will be true for CentOS 8.

Reply:
Done, thanks @abbra